### PR TITLE
Feature/73 application date

### PR DESCRIPTION
**Pull request by RohitGarudadri**
### Tickets:

- HCK-73

### List of changes:

- Prevented creating applications past the deadline date of Jan 3rd 2020, 11:59:59PM EST.

### Type of change:

Please delete options that aren't relevant.

- [x] New feature (non-breaking change which adds functionality)

### How did you do this?

- Check if Date.now() is below specified date where required

### How to test:

Try changing the date value from 1578070799000 to 1278070799000 and check that the appropriate things render.

### PR Checklist:

- [x] Merged `develop` branch (before testing)
- [x] Linted my code locally
- [x] Listed change(s) in the Changelog
- [x] Tested all links in project relevant browsers
- [x] Tested all links on different screen sizes
- [x] Referenced all useful info (issues, tasks, etc)

### Screenshots:
This is what appears if it is past the deadline: 
Screen Shot 2020-01-01 at 4 08 23 PM

### CODE REVIEW

**Comment by loreina** [comment]
can you update file changes without the code formatting?

**Comment by RohitGarudadri** [comment]
Try hiding the whitespace changes and reload to see what I actually changed

File: src/config/constants.ts [hunk 1]
```diff
@@ -49,6 +49,8 @@ export const LAST_NAME_LABEL = 'Last Name';
 // Status management
 export const NONE_STATUS_TEXT =
   'You’re all set! Ready to start your application?';
+export const DEADLINE_STATUS_TEXT =
```
Comment (loreina) [inline src/config/constants.ts]: I don't think this is a status, can we move this under Application management constants?

File: src/features/Application/ManageApplicationContainer.tsx [hunk 2]
```diff
@@ -147,74 +149,72 @@ class ManageApplicationContainer extends React.Component<
...
-            paddingTop={'70px'}
-          >
-            {mode === ManageApplicationModes.CREATE ? 'Create' : 'Edit'} your
-            Application
-          </H1>
-          <FormDescription>{CONSTANTS.REQUIRED_DESCRIPTION}</FormDescription>
+    const { mode, hackerDetails, submitted, pageNumber, loaded } = this.state;
+    return loaded ? (
+      // If application creation deadline has passed or form is submitted, return user to the home page
+      (Date.now() > 1578070799000 && mode === ManageApplicationModes.CREATE) ||
```
Comment (loreina) [inline src/features/Application/ManageApplicationContainer.tsx]: what's this date number? can you write a comment for what date it is for future reference?

File: src/features/Status/StatusPage.tsx [hunk 3]
```diff
@@ -63,25 +63,38 @@ class StatusPage extends React.Component<IStatusPageProps, {}> {
                     <Button type="button">View/Edit Application</Button>
                   </LinkDuo>
                 </Flex>
+              ) : Date.now() < 1578070799000 ? (
```
Comment (loreina) [inline src/features/Status/StatusPage.tsx]: same for here, can you comment the date this number means and where you generated? might actually be good to make this number a constant in the constants file and import it

File: src/config/constants.ts [hunk 4]
```diff
@@ -64,6 +64,9 @@ export const WITHDRAWN_STATUS_TEXT =
 export const CHECKED_IN_STATUS_TEXT = 'You’re checked-in and ready to go!';
 
 // Application management
+export const APPLICATION_CLOSE_TIME = 1578070799000; // Jan 3, 2020 11:59:59PM EST
+export const DEADLINE_PASSED_LABEL =
+  'Unfortunately the deadline to create an application for this year has passed. We hope to see you apply next year, though!';
```
Comment (loreina) [inline src/config/constants.ts]: change to:
"Sorry, we are no longer accepting applications at this time."

**Pull request merged**